src/components/molecules/Modal/Modal.tsx

```diff
@@ -50,11 +50,11 @@ export const Modal: React.FC<ModalProps> = ({
     >
       <div
         className={
-          "max-h-screen relative px-4 w-full max-w-2xl h-full md:h-auto"
+          "max-h-screen relative px-4 w-full max-w-2xl h-full md:h-auto sm:max-w-lg sm:w-full"
         }
         ref={containerRef}
       >
-        <div className="relative bg-white rounded-lg shadow dark:bg-gray-700">
+        <div className="inline-block align-middle bg-white rounded-lg px-4 pt-5 pb-4 text-left overflow-hidden shadow-xl transform transition-all sm:my-8 sm:max-w-lg sm:w-full sm:p-6">
```

Comment:
These sausages would be more manageable (not make the code too wide) if we put them in `Modal.tailwind.ts` and would probably be best we move this component in the `src/components/admin` directory so it's not mistakenly used in the attendee side.

Reply:
[3469e91](https://github.com/sparkletown/sparkle/pull/2944/commits/3469e91693f840fa4d04232bc8c99b452b8f6b6d)